manifest_staging/charts/gatekeeper/templates/gatekeeper-controller-manager-deployment.yaml

```diff
@@ -119,6 +122,15 @@ spec:
         - containerPort: {{ .Values.controllerManager.healthPort }}
           name: healthz
           protocol: TCP
+        - containerPort: 8443
```

Comment:
looks like these are being duplicated

Reply:
it is indeed a problem caused by the upgraded version of kustomize